test/testinput/bufr_snow_adpsfc.yaml

```diff
@@ -0,0 +1,113 @@
+
+observations:
+  - obs space:
+      name: bufr
+
+      obsdatain: "./gdas.t06z.adpsfc.tm00.bufr_d"
+
+      mnemonicSets:
+        - mnemonics: [YEAR, MNTH, DAYS, HOUR, MINU, WMOB, WMOS]
+        - mnemonics: [CLAT, CLON]
+        - mnemonics: [SELV, PRES, TMDB, TMDP, TOSD]
+
+      exports:
+        variables:
+          timestamp:
+            datetime:
+              year: YEAR
+              month: MNTH
+              day: DAYS
+              hour: HOUR
+              minute: MINU
+
+          longitude:
+            mnemonic: CLON
+
+          latitude:
+            mnemonic: CLAT
+
+          pressureAir:
+            mnemonic: PRES
+
+          stationElevation:
+            mnemonic: SELV
+
+          temperatureAir:
+            mnemonic: TMDB
+
+          temperatureDewpoint:
+            mnemonic: TMDP
+        
+          SnowDepth:
+            mnemonic: TOSD
+
+    ioda:
+      backend: netcdf
+      obsdataout: "../testoutput/gdas.t06z.adpsfc_snow.tm00.nc"
+
+      dimensions:
+        - name: "nlocs"
+          size: variables/latitude.nrows
+
+      variables:
+        - name: "datetime@MetaData"
+          source: variables/timestamp
+          dimensions: [ "nlocs" ]
+          longName: "Datetime"
+          units: "datetime"
+
+        - name: "latitude@MetaData"
+          source: variables/latitude
+          dimensions: ["nlocs"]
+          longName: "LATITUDE (COARSE ACCURACY)"
+          units: "degrees_north"
+          range: [-90, 90]
+
+        - name: "longitude@MetaData"
+          source: variables/longitude
+          dimensions: ["nlocs"]
+          longName: "LONGITUDE (COARSE ACCURACY)"
+          units: "degrees_east"
+          range: [-180, 180]
+
+        - name: "stationElevation@MetaData"
+          coordinates: "longitude latitude"
+          source: variables/stationElevation
+          dimensions: ["nlocs"]
+          longName: "HEIGHT OF STATION"
+          units: "Meter"
+
+        - name: "pressureAir@ObsValue"
+          coordinates: "longitude latitude"
+          source: variables/pressureAir
+          dimensions: ["nlocs"]
+          longName: "PRESSURE"
+          units: "Pa"
+          range: [90000, 105000] 
+
+        - name: "temperatureAir@ObsValue"
+          source: variables/temperatureAir
+          dimensions: ["nlocs"]
+          longName: "TEMPERATURE/DRY BULB TEMPERATURE"
+          units: "K"
+          range: [230, 315]
+
+        - name: "temperatureDewpoint@ObsValue"
+          coordinates: "longitude latitude"
+          source: variables/temperatureDewpoint
+          dimensions: ["nlocs"]
+          longName: "DEW POINT TEMPERATURE"
+          units: "K"
+          range: [200, 315]
+      
+        - name: "SnowDepth@ObsValue"
+          coordinates: "longitude latitude"
+          source: variables/SnowDepth
+          dimensions: ["nlocs"]
+          longName: "TOTAL SNOW DEPTH"
+          units: "METERS"
```

Comment:
why all caps? "Meter" is used above

Reply:
If you're referring to the "longName", this was the "name" of the variable used in the bufr file. 
I'll check again the units for Snow Depth again and respond to this with whether or not it needs to be changed

Reply:
Units, Meter vs METERS.

Reply:
They should both be "METERS". I'll edit, commit, and push again.

Reply:
well this might be a question for @gthompsnJCSDA (or others) on which to use. My guess is the units/names should match the output IODA conventions and not the input BUFR. All caps looks very 1985 to me. But I'm not sure about how the existing YAML files already in this repo are defined so maybe this is fine for now and is a problem to fix later.

Reply:
The IODA conventions document that I have says they should be a lowercase "m" so it should probably be that, but I'll wait to change, commit, and push we decide what to do.